src/styles/_catalog.scss

```diff
@@ -1,8 +1,22 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  column-gap: 48px;
+  justify-content: center;
+
+  grid-template-columns: 200px;
+
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, 200px);
```

Comment:
more correct to take out duplicate values in variables, because if you need, for example, to change the width of the card, then it is enough to change the value in the variable itself, and not change it throughout the code